barretenberg/cpp/src/barretenberg/stdlib/commitment/pedersen/pedersen.test.cpp

```diff
@@ -1,429 +1,291 @@
-#include "barretenberg/crypto/pedersen_commitment/pedersen.hpp"
```

Comment:
all of the code in here is also commented out, we can make an issue for this. Currently commitment is being tested indirectly through hash, though it would be good to have tests for commitment to document edge cases like when the input is all zeroes